setup.bat

```diff
@@ -0,0 +1,17 @@
+@ECHO OFF
+
+REM Install client dependencies
+call npm install grpc google-protobuf @grpc/proto-loader
+REM Remove modules (virtual environment folder)
+del modules
+REM Install virtualenv
+python -m pip install virtualenv --user
+REM Initialize modules as virtual environment
+virtualenv modules
```

Comment:
I get the error:
```
'virtualenv' is not recognized as an internal or external command,
operable program or batch file.
```
[Not sure if you've looked at this](https://stackoverflow.com/questions/35950740/virtualenv-is-not-recognized-as-an-internal-or-external-command-operable-prog)